pkg/config/config_test.go

```diff
@@ -439,3 +442,85 @@ func TestGetOSSourcePath(t *testing.T) {
 		})
 	}
 }
+
+func TestDeleteConfigDirIfEmpty(t *testing.T) {
+
+	t.Run("empty config dir", func(t *testing.T) {
```

Comment:
Would it be possible to convert this those test to use a similar system of a table drive tests as in all other tests?

Reply:
No actually, I tried that. The check and setup is different for both approach